cypress/support/commands.ts

```diff
@@ -103,6 +103,12 @@ Cypress.Commands.add("drupalCron", () => {
   cy.drupalLogout();
 });
 
+Cypress.Commands.add("drupalLoginAndVisit", (url: string) => {
+  cy.clearCookies();
+  cy.drupalLogin();
+  cy.visit(url);
+});
```

Comment:
I have to admit I am not too keen on this function.

1. It clears cookies which `drupalLogin()` does not. If this is necessary should we then move it to the `drupalLogin()` command?
2. It should be possible to chain `drupalLogin()` with `visit()` which should actually be shorter and more flexible than `drupalLoginAndVisit()`.